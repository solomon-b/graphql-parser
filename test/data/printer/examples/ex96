[ DefinitionTypeSystem
    ( TypeSystemDefinitionType
        ( STDef
            ( ScalarTypeDefinition
                { _stSpan = Span
                    { _start = AlexSourcePos
                        { _line = 0
                        , _col = 1
                        }
                    , _end = AlexSourcePos
                        { _line = 0
                        , _col = 68
                        }
                    }
                , _stDescription = Nothing
                , _stName = "UUID"
                , _stDirectives = Just
                    ( Directive
                        { _dSpan = Span
                            { _start = AlexSourcePos
                                { _line = 0
                                , _col = 13
                                }
                            , _end = AlexSourcePos
                                { _line = 0
                                , _col = 68
                                }
                            }
                        , _dName = "specifiedBy"
                        , _dArguments = Just
                            ( Arguments
                                { argSpan = Span
                                    { _start = AlexSourcePos
                                        { _line = 0
                                        , _col = 26
                                        }
                                    , _end = AlexSourcePos
                                        { _line = 0
                                        , _col = 68
                                        }
                                    }
                                , argArguments = fromList
                                    [
                                        ( "url"
                                        , VString
                                            ( Span
                                                { _start = AlexSourcePos
                                                    { _line = 0
                                                    , _col = 31
                                                    }
                                                , _end = AlexSourcePos
                                                    { _line = 0
                                                    , _col = 68
                                                    }
                                                }
                                            ) "https://tools.ietf.org/html/rfc4122"
                                        )
                                    ]
                                }
                            )
                        } :| []
                    )
                }
            )
        )
    )
]