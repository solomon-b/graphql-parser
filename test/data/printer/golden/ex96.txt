scalar UUID @specifiedBy(url: "https://tools.ietf.org/html/rfc4122")
